qemu/tests/floppy.py

```diff
@@ -193,7 +191,7 @@ def test(self):
             self.srchost = params["hosts"][0]
             self.dsthost = params["hosts"][1]
             self.is_src = params["hostid"] == self.srchost
-            self.mig = self.mig_type(test, params, env, False, )
+            self.mig = self.mig_type(test, params, env, True, )
```

Comment:
I know it, I mean when storage.preprocess_images called, how to ensure OS image file not be recreate?